lib/dash/content_protection.js

```diff
@@ -92,22 +92,22 @@ shaka.dash.ContentProtection.Element;
  * @private
  */
 shaka.dash.ContentProtection.defaultKeySystems_ = new Map()
-    .set('urn:uuid:1077efec-c0b2-4d02-ace3-3c1e52e2fb4b',
-         'org.w3.clearkey')
-    .set('urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed',
-         'com.widevine.alpha')
-    .set('urn:uuid:9a04f079-9840-4286-ab92-e65be0885f95',
-         'com.microsoft.playready')
-    .set('urn:uuid:f239e769-efa3-4850-9c16-a903c6932efb',
-         'com.adobe.primetime');
+  .set('urn:uuid:1077efec-c0b2-4d02-ace3-3c1e52e2fb4b',
```

Comment:
Formatting does not follow the style guide. The ".set" needs to be indented two tabs (four spaces). Then wrapped param needs to either be indented another two tabs (four spaces) or be inline with the original start of the parameters.

Reply:
Eslint is not currently enforcing whitespace and indentation rules.  I thought we had an open bug about this, but it seems we do not.

So right now, there is no automated feedback on these style guide issues.

The sad thing is that we will not be 100% in line with the Google style guide once we get eslint enforcing these rules, because eslint's indentation system can't accurately implement the Google style guide.  So we will wind up with something that's close enough and stick to it.  (In v2.6.)

In the mean time, @tylerdaines, please avoid changing any existing indentation patterns.  @vaage, please be flexible with minor indentation problems since there is no way for anyone to automate catching them.  We will need to clean up many, many pre-existing indentation problems once we turn on indentation checks in eslint, even in cases where we /do/ conform to the style guide already.  So it's not worth being strict about right now.